src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -29,6 +29,9 @@
 	    		component.set("v.householdRecTypeId", settingsNoPrefix.Household_Addresses_RecType__c);
 		        component.set("v.studentRecTypeId", settingsNoPrefix.Student_RecType__c);
 		        component.set("v.facultyRecTypeId", settingsNoPrefix.Faculty_RecType__c);
+				component.set("v.affiliationRoleMapValue",settingsNoPrefix.Affl_ProgEnroll_Role_Map__c);
```

Comment:
Yes, please verify that indentations are 4 spaces, and if we could avoid passing in method calls & ternaries at part of if conditions or other method calls that would help for readability.

Reply:
Also, I had some of our lightning gurus look at the components and they mentioned we really should be leveraging the "lighting" namespace with components rather than leveraging the "ui" namespaces. (e.g. lightning:input vs ui:input, etc.)